06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md

```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: the repository for your proposed project and at least one `Pull Request`
+- **With**: *independently*
+
+#### Link
+- [Repository](https://github.com/andela-angene/inverted-index)
```

Comment:
Please ensure to clearly outline the features of the application in the ReadMe and how to use it.